src/manage_users.py

```diff
@@ -0,0 +1,201 @@
+#!/usr/bin/env python3
+
+import argparse
+import getpass
+import os
+import sys
+
+from flask_security import Security
+from flask_sqlalchemy import SQLAlchemy
+
+from authenticate_app import create_db_interface
+from helperFunctions.config import load_config
+from version import __VERSION__
+from web_interface.frontend_main import WebFrontEnd
+
+
+def setup_argparse():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('-v', '--version', action='version', version='FACT User Management (FACTUM) {}'.format(__VERSION__))
+    parser.add_argument('config_file', help='path to fact config file')
```

Comment:
Thought about it. Might introduce a false idea of safety..
I think since this is not a comfort feature, the user should be as explicit as possible.
I for example do not use the main.cfg.

Reply:
I would prefer a default value for config path as well.